github/actions_runners.go

```diff
@@ -214,6 +214,50 @@ func (s *ActionsService) CreateOrganizationRegistrationToken(ctx context.Context
 	return registrationToken, resp, nil
 }
 
+// CreateEnterpriseRegistrationToken creates a token that can be used to add a self-hosted runner to an enterprise.
+//
+// GitHub API docs: https://docs.github.com/en/rest/reference/enterprise-admin#create-a-registration-token-for-an-enterprise
+func (s *ActionsService) CreateEnterpriseRegistrationToken(ctx context.Context, enterprise string) (*RegistrationToken, *Response, error) {
+	u := fmt.Sprintf("enterprises/%v/actions/runners/registration-token", enterprise)
+
+	req, err := s.client.NewRequest("POST", u, nil)
+	if err != nil {
+		return nil, nil, err
+	}
+
+	registrationToken := new(RegistrationToken)
+	resp, err := s.client.Do(ctx, req, registrationToken)
+	if err != nil {
+		return nil, resp, err
+	}
+
+	return registrationToken, resp, nil
+}
+
+// ListEnterpriseRunners lists all the self-hosted runners for an enterprise.
+//
+// GitHub API docs: https://docs.github.com/en/rest/reference/enterprise-admin#list-self-hosted-runners-for-an-enterprise
+func (s *ActionsService) ListEnterpriseRunners(ctx context.Context, enterprise string, opts *ListOptions) (*Runners, *Response, error) {
+	u := fmt.Sprintf("enterprises/%v/actions/runners", enterprise)
+	u, err := addOptions(u, opts)
+	if err != nil {
+		return nil, nil, err
+	}
+
+	req, err := s.client.NewRequest("GET", u, nil)
```

Comment:
We support this one too:

https://github.com/google/go-github/blob/master/github/enterprise_actions_runners.go#L37

Reply:
oh sorry I did not check that there are another file.. :)